modules/recommended-requirements-central-services.adoc

```diff
@@ -38,8 +39,8 @@ The following table lists the minimum memory and CPU values required to run Cent
 
 | < 25,000
 | < 5 users
-| 6 cores
-| 12 GiB
+| 2 cores
```

Comment:
I'm not able to find the analysis related to these changes. Maybe reducing core requirements would be ok, but I'm not sure about the memory requirement adjustment. I would rather keep it as it is.

Reply:
See above.